src/include/crowbar/complex.rb

```diff
@@ -741,18 +741,22 @@ def InitRPRepos(id)
               elsif url.include? "ks/dist/child"
                 @repos_location = "sm"
                 @remote_server_url = url.gsub(/(^.*)\/ks\/dist\/child\/.*/,"\\1")
+              elsif !url.empty?
+                @repos_location = "custom"
               end
-              break unless @repos_location == "custom"
+              break unless @repos_location == "local"
             end
-            break unless @repos_location == "custom"
+            break unless @repos_location == "local"
```

Comment:
The double `break` could be simplified with [`throw` and `catch`](http://ruby-doc.com/docs/ProgrammingRuby/html/tut_exceptions.html#S4).